hw/ip/rv_dm/doc/dv/index.md

```diff
@@ -24,95 +15,131 @@ applicable. Once done, remove this comment before making a PR. -->
-* Alerts ([`alert_esc_if`]({{< relref "hw/dv/sv/alert_esc_agent/README.md" >}}))
-* Devmode ([`pins_if`]({{< relref "hw/dv/sv/common_ifs" >}}))
+* [TileLink host interface]({{< relref "hw/dv/sv/tl_agent/README.md" >}}) for the config space (which only contains the alert register).
+* [TileLink host interface]({{< relref "hw/dv/sv/tl_agent/README.md" >}}) for the debug memory space (which only contains the alert register).
+* [TileLink device interface]({{< relref "hw/dv/sv/tl_agent/README.md" >}}) for the SBA interface that is used by the JTAG debugger to access the whole chip.
+* [JTAG interface]({{< relref "hw/dv/sv/jtag_agent/doc" >}}) to act as an external JTAG host.
+* [Alert and escalation interface]({{<relref "hw/dv/sv/alert_esc_agent/README.md" >}}) for the alert interface.
+  * The instantiation and hookup of this interface is done using the standardized common macro `` `DV_ALERT_IF_CONNECT`` defined in `hw/dv/sv/dv_utils/dv_macros.svh`.
+* RV_DM interface for driving / sampling the remaining DUT IOs
+  * Inputs driven / sampled: `lc_hw_debug_en_i`, `scanmode_i`, `scan_rst_ni`, `unavailable_i`
+  * Outputs sampled: `ndmreset_req_o`, `dmactive_o` & `debug_req_o`
 
 ### Common DV utility components
+
 The following utilities provide generic helper tasks and functions to perform activities that are common across the project:
 * [dv_utils_pkg]({{< relref "hw/dv/sv/dv_utils/README.md" >}})
+* [dv_lib_pkg]({{< relref "hw/dv/sv/dv_lib/README.md" >}})
+* [cip_lib_pkg]({{< relref "hw/dv/sv/cip_lib/doc" >}})
+* dv_base_reg_pkg
 * [csr_utils_pkg]({{< relref "hw/dv/sv/csr_utils/README.md" >}})
 
-### Compile-time configurations
-[list compile time configurations, if any and what are they used for]
-
-### Global types & methods
-All common types and methods defined at the package level can be found in
-`rv_dm_env_pkg`. Some of them in use are:
-```systemverilog
-[list a few parameters, types & methods; no need to mention all]
-```
 ### TL_agent
+
 The RV_DM testbench instantiates (already handled in CIP base env) [tl_agent]({{< relref "hw/dv/sv/tl_agent/README.md" >}}).
 This provides the ability to drive and independently monitor random traffic via the TL host interface into the RV_DM device.
 
-### Alert_agents
-RV_DM testbench instantiates (already handled in CIP base env) [alert_agents]({{< relref "hw/dv/sv/alert_esc_agent/README.md" >}}):
-[list alert names].
+### Alert_agent
+
+RV_DM testbench instantiates an [alert_agent]({{< relref "hw/dv/sv/alert_esc_agent/README.md" >}}).
+This is already done in the CIP base environment.
+It uses a string array of alert names to associate each instance of an alert signal. RV_DM exposes only a single alert signal, so this array is set to ["fatal_fault"].
+This alert is wired from the bus integrity logic.
+
 The alert_agents provide the ability to drive and independently monitor alert handshakes via alert interfaces in RV_DM device.
 
 ###  JTAG Agent
```

Comment:
```suggestion
### JTAG Agent
```
Seems like there is any extra space? I wonder if that will have any side effort on the rendered file.
Same as line 75.

Reply:
Fixed.